os/drivers/Makefile

```diff
@@ -79,7 +79,8 @@ include gpio$(DELIM)Make.defs
 include heapinfo$(DELIM)Make.defs
 include i2c$(DELIM)Make.defs
 include iotdev$(DELIM)Make.defs
-include kernel_test$(DELIM)Make.defs
+include kernel_test$(DELIM)kernel_test_codes$(DELIM)Make.defs
```

Comment:
I don't know this is right.
If you want to split the folders, you should make more folders. The compression and TZ is not a part of kernel.

Reply:
didn't get your point. can you please clear it more?
what you mean by make more folders? i already seprated kernel test files and network test files.

Reply:
You made belows
1. kernel_test/kernel_test_codes
2. kernel_test/network_test_codes

But `codes` is redundant (All of others are also codes), `test` is duplicated.
So, if you want to add sub-folders, I suggest `kernel` and `network`.
1. kernel_test/kernel
2. kernel_test/network

Reply:
corrected